content/blog/ListView-conditional-formatting.ru.adoc

```diff
@@ -18,7 +18,7 @@ In this post we’ll be adding a simple coloured border to the
 ''Industry'' field in the Accounts list view. We’ll end up with
 something similar to this:
 
-:imagesdir: ./../../images/en/community
+:imagesdir: ./../../../images/en/community
```

Comment:
I have been changing all the `imagesdir` paths to absolute, since there is no reason to make them relative, and it avoids many errors with the levels of `..`.

So I suggest (here and in other similar places) changing to 
`:imagesdir:/images/en/community`

The only links that need to stay relative are the internal links to pages, since this is necessary to keep the link stable across all languages.

Reply:
In this case I suggest to replace the `imagesdir` path to absolute in all *.adoc files where the relative path is used, ok?

Reply:
If you want to that for all files at once, go ahead, just be careful and make sure you get it right :-)

The only reason I haven't done it myself is lack of time...